solutions/advanced/Ligatures.mdx

```diff
@@ -0,0 +1,272 @@
+---
+id: Ligatures
+source: Kattis
+title: Ligatures
+author: Alexis Tao
+---
+
+### Explanation
+
+The problem involves counting the occurrences of specific pairs of characters (ligatures) in a given text corpus based on multiple sets of suggested ligatures. The primary challenge is efficiently handling large input sizes and ensuring that the counting process respects the rule that ligatures can't overlap.
+
+### Solution
+
+This solution to the problem uses SIMD (Single Instruction, Multiple Data) parallelism to efficiently count the occurrences of suggested ligatures within a large corpus:
+
+**Overview**
+1. **Preprocessing the Corpus**:
```

Comment:
the numbering is messed up - you probably want to add spaces before the hyphens to fix this

<img width="398" alt="Screenshot 2024-06-23 at 7 11 22 PM" src="https://github.com/cpinitiative/usaco-guide/assets/21228024/dc2b6921-af7d-4e95-ac50-96bf1c00b17d">

Reply:
you can check in https://usaco.guide/editor/